filament/src/Renderer.cpp

```diff
@@ -307,15 +307,14 @@ void FRenderer::renderJob(ArenaScope& arena, FView& view) {
             .hasContactShadows = scene.hasContactShadows()
     };
 
-    const ToneMappingConfig toneMappingConfig{
-            .toneMappingAsSubpass =
-                    toneMapping &&
+    const ColorGradingConfig colorGradingConfig{
+            .asSubpass =
```

Comment:
This needs to be `false` if post processing is disabled, because in that case, color-grading is also disabled. We don't want to take the subpass path during the color pass.
I think it would be better to keep the `tonemapping` boolean (renamed) in this function.